ui/app/serializers/kv/data.js

```diff
@@ -22,7 +22,6 @@ export default class KvDataSerializer extends ApplicationSerializer {
     return {
       ...payload,
       data: {
-        ...payload.data,
```

Comment:
I was a little confused by our data return here. See [question](https://github.com/hashicorp/vault/pull/21434/files#r1239085000) I posted to Chelsea (about the adapter but about the returned response).  What does removing payload.data effect here?

Reply:
This is a slightly different situation than the one linked above. If you look at line 21 `payload.data` is assigned the `const data` which below is added to the key `secret_data`. I removed `payload.data` to organize the response a bit so that the secret data didn't exist twice, now it just lives within `secret_data`

Reply:
Got it, thank you for the clarification!